uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/io/HabitsCSVExporter.kt

```diff
@@ -77,7 +77,7 @@ class HabitsCSVExporter(
     }
 
     private fun sanitizeFilename(name: String): String {
-        val s = name.replace("[^ a-zA-Z0-9\\._-]+".toRegex(), "")
+        val s = name.replace("[^ a-zA-Z0-9._-]+".toRegex(), "")
```

Comment:
Why is this needed?

Reply:
Not needed, it's just a redundant escape. Shall I revert this change?